dao/src/main/kotlin/queries/GetAnalyzerConfigurationForAnalyzerRun.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.dao.queries
+
+import org.eclipse.apoapsis.ortserver.dao.Query
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.AnalyzerConfigurationsPackageManagerConfigurationsTable
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.AnalyzerConfigurationsTable
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.AnalyzerConfigurationsTable.allowDynamicVersions
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.AnalyzerConfigurationsTable.disabledPackageManagers
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.AnalyzerConfigurationsTable.enabledPackageManagers
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.AnalyzerConfigurationsTable.skipExcluded
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.PackageManagerConfigurationOptionsTable
+import org.eclipse.apoapsis.ortserver.dao.repositories.analyzerrun.PackageManagerConfigurationsTable
+import org.eclipse.apoapsis.ortserver.model.runs.AnalyzerConfiguration
+import org.eclipse.apoapsis.ortserver.model.runs.PackageManagerConfiguration
+
+import org.jetbrains.exposed.sql.selectAll
+
+/**
+ * A query to get the [AnalyzerConfiguration] for a given [analyzerRunId]. Returns `null` if the configuration is not
+ * found.
+ */
+class GetAnalyzerConfigurationForAnalyzerRun(
```

Comment:
Since we can expect a large number of classes, should we already start to group them in sub packages? E.g. an `analyzer` package for Analyzer-related queries.